manager-dashboard/app/components/TileServerInput/index.tsx

```diff
@@ -0,0 +1,201 @@
+import React from 'react';
+import {
+    useFormObject,
+    getErrorObject,
+    PartialForm,
+    Error,
+    SetValueArg,
+    ObjectSchema,
+    requiredStringCondition,
+    requiredCondition,
+} from '@togglecorp/toggle-form';
+
+import TextInput from '#components/TextInput';
+import RadioInput from '#components/RadioInput';
+import {
+    valueSelector,
+    labelSelector,
+    getNoMoreThanNCharacterCondition,
+} from '#utils/common';
+
+export type TileServerType = 'bing' | 'mapbox' | 'maxar_standard' | 'maxar_premium' | 'esri' | 'esri_beta' | 'sinergise' | 'custom';
```

Comment:
we can remove all references to `singergise`. imagery from Sinergise is not provided anymore